terraform/scripts/place_ssh_keys.sh

```
@@ -0,0 +1,46 @@
+export USER="devops"
+export KEYS_DIR="."
+BASTION_KEYS=()
+
+generate_keys() (
+  yes y | ssh-keygen -f link -N '' -t ed25519
+)
+
+wait_for_readiness () (
+  HOST="$1"
+  shift
+  while ! gcloud compute ssh "$USER"@"$HOST" --command=true $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)